docs/template_plugin/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
maybe we can all this file as `configure_tests.hpp` 
Because we don't tests RO properties here

Reply:
Any Ideas on how we can test RO tests using a map of properties?
Default value test can get RO and compare values with expected.
May be provide two maps  input parameters and expected configuration values

Reply:
> @ilya-lavrenov what can you say?

can rename to `configure_tests.hpp` ? or something with `configure `